Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -159,13 +159,14 @@ public void MoveDirectory(string source, string destination)
             catch (IOException)
             {
                 // Cross device move requires a copy
-                var directory = Directory.CreateDirectory(destination);
-                foreach (var file in directory.EnumerateFiles())
+                Directory.CreateDirectory(destination);
```

Comment:
Why is this necessary? Isn't that what it already does?

Reply:
`Directory.Move` does not handle the case where the source and destination are not on the same file system (like moving cross disks) and will throw an IOException.